src/parser/lessParser.ts

```diff
@@ -348,7 +469,13 @@ export class LESSParser extends cssParser.Parser {
 			}
 			return null;
 		};
-		while (this.accept(TokenType.Ident) || node.addChild(this._parseInterpolation() || this.try(delimWithInterpolation))) {
+		const accept = identRegex ?
+			(): boolean => this.acceptRegexp(identRegex) :
+			(): boolean => this.accept(TokenType.Ident);
+
+		while (accept() || 
+			node.addChild(this._parseInterpolation() || 
+			this.try(delimWithInterpolation))) {
```

Comment:
You can the conditions in the same line. It's hard to read what's inside the `while` because of no indentation.

Reply:
Fair enough. Or maybe for really long conditions, move them all to an indented form?

As in:

```js
while(
  condition() ||
  condition2()
) {
}
```
?

Reply:
Yeah that's better. `this.try` and `hasContent` on the same indent level was confusing to me.